lib/Chat/SystemMessage/Listener.php

```diff
@@ -382,6 +382,10 @@ public static function fixMimeTypeOfVoiceMessage($event): void {
 		$listener = Server::get(self::class);
 		$manager = Server::get(Manager::class);
 
+		$request = Server::get(IRequest::class);
+		if ($request->getParam('_route') === 'ocs.spreed.Recording.shareToChat') {
+			return;
+		}
```

Comment:
I need do this to prevent run this method and duplicate the system message

Reply:
This sounds illegal 🙈

Please raise a follow up tech-debt issue to replace this with something better.
E.g. we could have a "state" variable in the RecordingService about this, that skips it and afterwards resets.